release-notes.md

```diff
@@ -1,8 +1,30 @@
 # Release Notes
 
-## 3.1
+## 3.2.0
 
-### Declarative Configuration Support for Core Framework v3.8
+### Prompt before configuration overwrites
+
+This new feature warns and prompts users when uploading a configuration if the configuration being overwritten is not the last known configuration. These warnings are comparable to a database write conflict. Previous versions of medic-conf would upload the new configuration overwriting whatever was there, resulting in users accidentally wiping out someone elses changes or changes made through the admin console.
+
+To detect this the cli will generate two files - `.snapshots/remote.json` and `.snapshots/local.json`. These store the last known configuration on the server. You should commit the `remote.json` with your configuration changes, but add the `local.json` to your .gitignore because everyone's localhost is different. That way if anyone changes the configuration on the server using medic-conf or the App Management webapp without committing the `remote.json` file to your config repo you will be notified about the risk and prompted to overwrite or cancel.
```

Comment:
Better to stay consistent with how we refer to medic-conf, so I'd suggest we change `the cli` to  `medic-conf`. Also made some other clarifications throughout the paragraph.

```suggestion
To detect this, medic-conf will generate two files - `.snapshots/remote.json` and `.snapshots/local.json`. These store the last known configuration on the server. The `remote.json` file should be commited with your configuration changes, and `local.json` should be added to the .gitignore since everyone's local environment is different. If changes are made to the configuration on the server using medic-conf or the App Management webapp without committing the `remote.json` file to your config repo, and then subsequently pulling the changes, then any attempt to upload configuration changes will be notified about the risk and prompted to overwrite, or cancel.
```

Reply:
Note, the changes aren't highlighted very well here!